Automattic/wp-calypso
# Fix the build on Windows

The build is currently broken on Windows - reported at #25187.  We're launching some `npm` scripts with the `ENV_VAR=value command` syntax, which is not compatible with Windows.

This PR fixes the things I see in `package.json` that will break the build on Windows, and also removes the `env` npm script to try to slightly reduce the level of `npm` process-launching insanity.

I don't have Windows so I would appreciate some help testing there.

### To test

Run a full Calypso build.

You can also run the affected build commands directly and verify that they still work:

```
npm run build-devdocs:components-usage-stats
npm run build-devdocs:index
npm run build-server
npm run build-client
npm run poststart
```
